src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/ProjectCreation.kt

```diff
@@ -58,6 +60,13 @@ class ProjectCreationWizard : Wizard() {
         add(SelectLanguage::class)
         add(SelectCollection::class)
 
+        creationViewModel.creationCompletedProperty.onChange {
+            if (it) {
+                runLater {
+                    close()
+                }
+            }
+        }
```

Comment:
Does this still work if the user tries the wizard again? I didn't see where the property is reset to false

Reply:
i'm not certain. i'll check